Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -1193,7 +1189,7 @@ - (void)_updateRuntimeProtocolConformanceIfNeeded
             unsigned methodCount = 0;
             using MethodDescriptionList = objc_method_description[];
             auto methods = std::unique_ptr<MethodDescriptionList, WTF::SystemFree<MethodDescriptionList>> {
-                protocol_copyMethodDescriptionList(protocol, NO, YES, &methodCount)
+                protocol_copyMethodDescriptionList(protocol, YES, YES, &methodCount)
```

Comment:
Did the protocol specification change? Or should these always have been required methods?

Reply:
Ah, they're now required in the new text input protocol (whereas they weren't before). I went ahead and adjusted this to match that.